app/backend/src/couchers/models.py

```diff
@@ -654,3 +654,60 @@ class InitiatedUpload(Base):
     @hybrid_property
     def is_valid(self):
         return (self.created <= func.now()) & (self.expiry >= func.now())
+
+
+class BackgroundJobType(enum.Enum):
```

Comment:
Is it time to create a models folder? :P

Reply:
Great idea! I just created a draft for it: https://github.com/Couchers-org/couchers/pull/482